challenges/classes.js

```diff
@@ -1,7 +1,37 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker2{
+    constructor(length, width, height) {
+    this.length = length;
+    this.width = width;
+    this.height = height;
+  }
 
+  volume(){
+    return this.length * this.width * this.height
+  }
+  surfaceArea(){
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height)
+  }
+}
+  
+  let cuboid2 = new CuboidMaker2 (4,5,5)
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
+console.log(cuboid.surfaceArea()); // 130
 
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
+class CubeMaker extends CuboidMaker2{
+    constructor(side){
+        super(side,side,side)
```

Comment:
Are you passing side as an object?
instead, you will have side.length, side.width etc

Alternatively, you may pass the dimensions as separate entries.